boot/rtl/blrtl.cmake

```diff
@@ -16,11 +16,8 @@ list(APPEND SOURCE
     # ${NTOS_RTL_SOURCE_DIR}/assert.c   ## Requires a local implementation.
     ${NTOS_RTL_SOURCE_DIR}/bitmap.c
     # ${NTOS_RTL_SOURCE_DIR}/bootdata.c ## Requires a local implementation.
-    ${NTOS_RTL_SOURCE_DIR}/compress.c
-    ${NTOS_RTL_SOURCE_DIR}/crc32.c
```

Comment:
Please reintroduce, as these will be needed to support hibernation file. (And crc32 for other things as well.)